src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -177,13 +207,30 @@ doAccountLines(RPC::JsonContext& context)
                 startHint,
                 reserve,
                 [&visitData](std::shared_ptr<SLE const> const& sleCur) {
+                    bool ignore = false;
+                    if (visitData.ignoreDefault)
+                    {
+                        if (sleCur->getFieldAmount(sfLowLimit).getIssuer() ==
+                            visitData.accountID)
+                            ignore =
+                                !(sleCur->getFieldU32(sfFlags) & lsfLowReserve);
+                        else
+                            ignore = !(
+                                sleCur->getFieldU32(sfFlags) & lsfHighReserve);
+                    }
+
                     auto const line =
                         RippleState::makeItem(visitData.accountID, sleCur);
                     if (line != nullptr &&
                         (!visitData.hasPeer ||
                          visitData.raPeerAccount == line->getAccountIDPeer()))
                     {
-                        visitData.items.emplace_back(line);
+                        if (!ignore)
+                            visitData.items.emplace_back(line);
+
+                        visitData.lastFound = line;
+                        visitData.foundCount++;
```

Comment:
I think this would be somewhat more readable if `ignore` was `include`.  So it would look something like:
```
bool include = true;
if (visitData.ignoreDefault)
{
    if (sleCur->getFieldAmount(sfLowLimit).getIssuer() ==
       visitData.accountID)
       include = sleCur->getFieldU32(sfFlags) & lsfLowReserve;
    else
        ignore = sleCur->getFieldU32(sfFlags) & lsfHighReserve;
}
[...]
{
    if (include)
        visitData.items.emplace_back(line);
[...]